front-end/src/components/CustomNode/GraphView.js

```diff
@@ -99,44 +99,43 @@ export default class GraphView extends React.Component {
 
       if (this.state.columnCount < 1) {
         return (
-          <Modal show={this.props.show} onHide={this.props.toggleShow} centered>
+          <Modal show={this.props.show} onHide={this.props.toggleShow} centered
+             onWheel={e => e.stopPropagation()}>
           <Modal.Header>
               <Modal.Title><b>{this.props.node.options.name}</b> View</Modal.Title>
           </Modal.Header>
           <Modal.Body>
           Loading the data might take a while depending on how big the data is.
           </Modal.Body>
           <Modal.Footer>
-              <Button variant="secondary" onClick={this.onClose}>Accept</Button>
+              <Button variant="secondary" onClick={this.onClose}>Cancel</Button>
               <Button variant="secondary" onClick={this.load}>Load</Button>
           </Modal.Footer>
           </Modal>
         );
       }
 
       return (
-              <Modal show={this.props.show} onHide={this.props.toggleShow} centered>
-              <Modal.Header>
+          <Modal show={this.props.show} onHide={this.props.toggleShow} centered
+              onWheel={e => e.stopPropagation()}>
```

Comment:
I tried this and didn't work, does it work locally for you?

Reply:
Hmmm, yeah works for me on all modals

Reply:
oh, that's good. Maybe I did it wrong because when I did, it stopped propagating the scroll event, but also it stopped scrolling the data

Reply:
Haha well let's see if it works for anyone else first

Reply:
I just tried it and it worked as expected